keyboards/treadstone48/info.json

```diff
@@ -0,0 +1,182 @@
+[
```

Comment:
This file should be deleted, as ` keyboards/treadstone48/rev1/info.json` will take precedence.